flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/SqlGatewayRestEndpoint.java

```diff
@@ -47,7 +57,43 @@ public SqlGatewayRestEndpoint(Configuration configuration, SqlGatewayService sql
     @Override
     protected List<Tuple2<RestHandlerSpecification, ChannelInboundHandler>> initializeHandlers(
             CompletableFuture<String> localAddressFuture) {
-        return new ArrayList<>();
+        List<Tuple2<RestHandlerSpecification, ChannelInboundHandler>> handlers =
+                new ArrayList<>(32);
```

Comment:
Unless there's a good reason, we usually don't set the initial capacity of a collection.

Reply:
Done.